### DISCUSSION TITLE
Adding storage to existing storage

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#4848] [Q&A] [answered]
If I am now using Immich with a 2TB disk and in the future I would like to add another 4TB, how can I make Immich also use the 4TB disk without rebuilding the entire system (so 2TB + 4TB)?
I am aware that there are libraries but this does not provide an answer as far as I understood

This title is not good. If anyone has a good idea for a different title so that others can find it easily, I would be happy to change it

**Comment by bo0tzz** [comment, marked as answer]
This isn't something for Immich to do, but rather something that you would (usually) set up through your OS. There's many different ways such as RAID, ZFS, LVM, etc. Which one to pick depends on a lot of factors about your setup and such.

**Reply by aviv926** [reply]
Thanks, I actually haven't installed Immich definitively on my computer yet
All I did was test it on a temporary computer to see if it suited my needs
Now I'm sure this is what I'm looking for, I read a little about RAID 0 and RAID 1, probably what I need is RAID 0 with backups once in a while to be sure everything is kept safe
I thought about RAID 10 but it doesn't seem like Windows 10 supports it

What do you think? Is this a good enough setup?
